build.gradle

```diff
@@ -235,9 +238,11 @@ dependencies {
     implementation 'org.controlsfx:controlsfx:11.2.1'
 
     implementation 'org.jsoup:jsoup:1.17.2'
-    implementation 'com.konghq:unirest-java:3.14.5'
+    implementation 'com.konghq:unirest-java-core:4.4.0'
+    implementation 'com.konghq:unirest-modules-gson:4.4.0'
+    implementation 'org.apache.httpcomponents.client5:httpclient5:5.3.1'
 
-    implementation 'org.slf4j:slf4j-api:2.0.13'
+    implementation "org.slf4j:slf4j-api:2.0.13"
```

Comment:
I think, all dependencies should be in ' not in "

Double check with dependabot. They IMHO can do only '

Reply:
Did not find anything int he docs, changed it back to single quote to be consistent